.gitignore

```diff
@@ -2,7 +2,7 @@
 *.iml
 *.py[co]
 .*.sw[a-z]
-.coverage
+.coverage*
```

Comment:
darn, reverted. coverage-- for littering .coverage$hostname files all over my repo though.